bff-node/src/config/database.ts

```diff
@@ -0,0 +1,16 @@
+import { Sequelize } from 'sequelize-typescript';
+import { HelloWorld } from '../models/HelloWorld';
+
+type SupportedDBDialect = 'mysql' | 'postgres' | 'sqlite' | 'mariadb' | 'mssql' | 'mariadb';
+
+const database = process.env.DATABASE_NAME ?? '';
+const username = process.env.DATABASE_USERNAME ?? 'root';
+const password = process.env.DATABASE_PASSWORD ?? 'root';
+const host = process.env.DATABASE_HOST ?? 'localhost';
+const dialect = (process.env.DATABASE_DIALECT ?? 'mysql') as SupportedDBDialect;
```

Comment:
Nice that you have installed an ORM for integration with a database, but do you know if a database is needed within the project shortly? If not this will be a dependency that needs maintenance over time and it not in use. 😅

Reply:
If you decide to keep it, consider we should consider if MySQL is the correct fallback dialect or not.